release-notes.md

```diff
@@ -0,0 +1,18 @@
+======================================================
```

Comment:
What is this line?

Reply:
Maybe we should have a title here instead:
# Oak Release Notes

Reply:
This line I thought to use for the future separation between releases. Like the future release notes should come above this line. I wanted to put it there so future updater won't forget to separate between releases. But if it is confusing I can remove it.